EIPS/eip-2464.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: 2464
+title: "eth/65: transaction annoucements and retrievals"
+author: Péter Szilágyi <peterke@gmail.com>, Gary Rong <garyrong0905@gmail.com>
+discussions-to: https://github.com/ethereum/EIPs/issues/2465
+status: Draft
+type: Standards Track
+category: Networking
+created: 2020-01-13
+requires: 2365
+---
+
+## Abstract
+
+The `eth` network protocol has two ways to propagate a newly mined block: it can be broadcast to a peer in its entirety (via `NewBlock (0x07)` [in `eth/64` and prior](https://github.com/ethereum/devp2p/blob/master/caps/eth.md)) or it can be announced only (via `NewBlockHashes (0x01)`). This duality allows nodes to do the high-bandwidth broadcasting (10s-100s KB) for a logarithmic number of peers; and the low-bandwidth announcing (10s-100s B) for the remaining linear number of peers. The logarithmic broadcast is enough to reach all well connected nodes, but the linear announce is needed to get across degenerate topologies. This works well.
```

Comment:
perhaps not logarithmic number of peers, but square root number of peers?

Reply:
Does it matter much though? Curious, no idea.